### PR TITLE
Edit student form 3

### PR DESCRIPTION
**Pull request by PaulDuxblah**

### CODE REVIEW

File: routes/student.route.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ studentRoutes.route('*').all(checkIfAuthenticated, function (req, res, next) {
 
 // POST
 studentRoutes.route('/').post(function (req, res) {
+  console.log('post');
```
Comment (jcervelle) [inline routes/student.route.js]: debug console.log

File: src/app/app.module.ts [hunk 2]
```diff
@@ -11,8 +11,9 @@ import { AdminService } from './services/admin.service';
 import { StudentService } from './services/student.service';
 import { LoggedComponent } from './logged/logged.component';
 import { StudentComponent } from './logged/student/student.component';
-import { AddComponent } from './logged/student/add/add.component';
+import { FormComponent } from './logged/student/form/form.component';
 import { AllComponent } from './logged/student/all/all.component';
+import { AddComponent } from './logged/student/add/add.component';
```
Comment (jcervelle) [inline src/app/app.module.ts]: Name of the import not explicit, could be redundant. StudentAddComponent could be a better choice.

File: src/app/app.module.ts [hunk 3]
```diff
@@ -11,8 +11,9 @@ import { AdminService } from './services/admin.service';
...
 import { EditComponent } from './logged/student/edit/edit.component';
```
Comment (jcervelle) [inline src/app/app.module.ts]: Same as last one.

File: src/app/app.module.ts [hunk 4]
```diff
@@ -11,8 +11,9 @@ import { AdminService } from './services/admin.service';
 import { StudentService } from './services/student.service';
 import { LoggedComponent } from './logged/logged.component';
 import { StudentComponent } from './logged/student/student.component';
-import { AddComponent } from './logged/student/add/add.component';
+import { FormComponent } from './logged/student/form/form.component';
 import { AllComponent } from './logged/student/all/all.component';
```
Comment (jcervelle) [inline src/app/app.module.ts]: Same as next one.

File: src/app/logged/student/student.component.ts [hunk 5]
```diff
@@ -4,7 +4,6 @@ import { StudentService } from '../../services/student.service';
 import { AddComponent } from './add/add.component';
```
Comment (jcervelle) [inline src/app/logged/student/student.component.ts]: Name of the import not explicit, could be redundant. StudentAddComponent could be a better choice.

File: src/app/logged/student/student.component.ts [hunk 6]
```diff
@@ -4,7 +4,6 @@ import { StudentService } from '../../services/student.service';
...
 import { AllComponent } from './all/all.component';
```
Comment (jcervelle) [inline src/app/logged/student/student.component.ts]: Same as last one.

File: src/app/logged/student/student.component.ts [hunk 7]
```diff
@@ -4,7 +4,6 @@ import { StudentService } from '../../services/student.service';
...
 import { EditComponent } from './edit/edit.component';
```
Comment (jcervelle) [inline src/app/logged/student/student.component.ts]: Same as last one.

File: src/app/app.module.ts (hunk 4 above)
Comment (PaulDuxblah) [reply src/app/app.module.ts]: You need some glasses mate
add and all

**Pull request merged**